src/core/qfieldcloudprojectsmodel.cpp

```diff
@@ -222,15 +222,29 @@ QVariantMap QFieldCloudProjectsModel::getProjectData( const QString &projectId )
   return data;
 }
 
-void QFieldCloudProjectsModel::refreshProjectsList( bool shouldRefreshPublic )
+void QFieldCloudProjectsModel::refreshProjectsList( bool shouldRefreshPublic, int projectFetchOffset )
```

Comment:
When projectFetchOffset is 0, reset model (so a project refresh removes projects that have disappeared on the cloud through a/ project deletion, b/ shared project revocation, c/ etc. :)

Reply:
Done.